articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,179 @@
+`Flutter` is an open-source UI software development kit that `Google` released in 2018. It uses Dart as its primary programming language. Flutter has led to the emergence of new software developers called `Flutter Developers`. 
+
+### Introduction
+
+Cross-platform applications enable companies to cut down on development costs. It eliminates the need to hire different software development teams. They also reduce the time and cost to manage different code bases by using a single code base to develop applications.  `Flutter developers` develop these cross-platform applications. 
```

Comment:
It eliminates the need ... --- are we still discussing cross-platform applications? If so, `it` is wrong.

Reply:
Could you rephrase 
`Flutter developers` develop these cross-platform applications.
To
Flutter allows developers to build cross-platform applications